src/posts/resize-text.md

```diff
@@ -103,9 +103,12 @@ To do a more thorough job, you should check the page at each of the zoom steps t
 
 
 ## Can I zoom everything instead of resizing text only?
+It depends on which legislation you have to follow. It's still a good idea to make sure that pages work well with text-only zoom since some people want only the text to be bigger. The usage has increased especially on mobile devices, and statistics from [appt.org](https://appt.org/en) (2024) show that more than a third of users on Android and iOS devices use this setting.
```

Comment:
>statistics from [appt.org](https://appt.org/en) (2024)

Is there a more specific URL we can link to here?

Reply:
Would this URL suffice? https://appt.org/en/stats
Since it covers both Android and iOS it would need to be general...

Reply:
Added the above link.